invenio_records_resources/services/records/service.py

```diff
@@ -27,12 +27,16 @@
 from .schema import ServiceSchemaWrapper
 
 
-class RecordService(Service):
-    """Record Service."""
+class RecordIndexerMixin:
+    """Mixin class to define record indexer."""
+
+    def __init__(self, config):
+        """Constructor.
+
+        :param config: A service configuration
+        """
+        self.config = config
```

Comment:
moderate: shouldn't this stay in the `RecordService`, or even be moved up to `Service`?

Reply:
It's already in the `Service`, I added it there as well as it needs the configuration to work, ideally it should not be used alone, but if done it would fail if the config is missing.